eq-author/cloudbuild.yaml

```diff
@@ -1,11 +1,11 @@
 # [START cloudbuild]
 steps:
-  - name: node
+  - name: node:$_NODE_VERSION
```

Comment:
Where do  you give this variable a value?

Reply:
GCP console.